src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -695,6 +697,38 @@ public synchronized Set<String> getStorageWorkers() throws IOException {
     return new HashSet<>(storageWorkerSet);
   }
 
+  @Override
+  public Map<String, Long> getWorkersStartTimeInEpochSecs(Set<String> workerNames)
+      throws IOException {
+    if (workerNames.isEmpty()) {
+      return Collections.emptyMap();
+    }
+    List<String> workerSet = client.call(jedis -> state.storageWorkers.mget(jedis, workerNames));
+
+    return workerSet.stream()
+        .filter(Objects::nonNull)
```

Comment:
why would you get null entries from a workerSet? (also calling it workerSet and being a list is a little confusing)

Reply:
Redis.hmget returns null if the field doesn't exist in the map. Also updated unit test to cover this case.

https://redis.io/commands/hmget/
```
For every field that does not exist in the hash, a nil value is returned. Because non-existing keys are treated as empty hashes, running HMGET against a non-existing key will return a list of nil values.
```

Renamed from workerSet to workerList, thanks!